test/test.js

```diff
@@ -458,6 +462,34 @@ function checkRefTestResults(browser, id, results) {
   });
 }
 
+function browserTestReportHandler(req, res) {
+  var parsedUrl = url.parse(req.url, true);
+  var pathname = parsedUrl.pathname;
+  if (pathname === '/browserTestReports' ||
+      pathname === '/browserWorkerTestReports') {
+
+    var map = libCoverage.createCoverageMap('window.__coverage__');
```

Comment:
`createCoverageMap` takes an object (= the coverage data) to construct the map, not a string. For documentation, see the source of `istanbul-lib-coverage`:

- https://github.com/istanbuljs/istanbuljs/blob/45936de1356b743dcf82df729bf3da895e180aea/packages/istanbul-lib-coverage/index.js#L32-L43
- https://github.com/istanbuljs/istanbuljs/blob/45936de1356b743dcf82df729bf3da895e180aea/packages/istanbul-lib-coverage/lib/coverage-map.js#L10-L35